### PR TITLE
dotCMS/core#24761 Fix Incorrect URLContentPage Title When Adding

### LINKED ISSUE #24761: Favorites Pages: Fix Incorrect URLContentPage Title When Adding
### Problem Statement
When adding a URLContentPage to favorites, the title is not correct and instead displays the title of the detail page instead of the content. 

### Steps to Reproduce
1. Edit a blog post page
2. Add the page to fav
3. See the page is "Blog Details"

### Acceptance Criteria
- The title of the URLContentPage page when added to favorites should be the title of the content and not the title of the detail page.

### External Links
N/A

### Assumptions & Initiation Needs
N/A

### Quality Assurance Notes & Workarounds
- Quality Assurance should verify the title of the URLContentPage is correct when added to favorites. 
- There is no workaround for this issue.

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
* When adding a URLContentPage to favorites, the title is not correct and instead displays the title of the detail page instead of the content.

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
image

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 @use "variables" as *;
 
 ::ng-deep {
+    .p-datatable .p-datatable-thead th.p-sortable-column:hover {
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss:4]: This should go in the global styles and also for not sortables header as well.

File: core-web/libs/dotcms-models/src/lib/dot-rendered-page.model.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export interface DotPageRenderParameters {
     canCreateTemplate: boolean;
     viewAs: DotEditPageViewAs;
     numberContents: number;
+    urlContentMap?: { title?: string };
```
Comment (fmontes) [inline core-web/libs/dotcms-models/src/lib/dot-rendered-page.model.ts:22]: The title is required here.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss (hunk 1 above)
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss:4]: not sortables columns does not has that bug, only sortables

Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss]: fix moved to global styles

File: core-web/libs/dotcms-models/src/lib/dot-rendered-page.model.ts (hunk 2 above)
Comment (alfredo-dotcms) [reply core-web/libs/dotcms-models/src/lib/dot-rendered-page.model.ts]: fixed

**Pull request merged**